locale/en.yml

```diff
@@ -604,9 +604,6 @@ en:
       last_started_at: Last Started At
       last_state: Last State
       last_sync_on: Last Sync Time
-      last_update_check: Last Update Check
```

Comment:
This still exists on miq_servers

Reply:
I'm removing all column updates from this PR. just focusing on the code